build/fuses/fuses.json5

```diff
@@ -2,5 +2,7 @@
   "_comment": "Modifying the fuse schema in any breaking way should result in the _version prop being incremented.  NEVER remove a fuse or change its meaning, instead mark it as removed with 'r'",
   "_schema": "0 == off, 1 == on, r == removed fuse",
   "_version": 1,
-  "run_as_node": "1"
+  "run_as_node": "1",
+  // Cookie Encryption is currently only supported on macOS and Windows
+  "cookie_encryption": "1"
```

Comment:
does this absolutely have to be a fuse? could it be a feature instead?

Reply:
@nornagon For codesigning reasons, yes IMO.  A feature flag is controlled via JS (and therefore not enforced via code signing), a fuse is enforced for code signing and for something security-related like cookie encryption I think it makes more sense to be a fuse

Reply:
I'm not sure this makes sense, if you're worried about an attack which modifies JS, then such an attack could easily steal cookies post-decryption. Protecting against JS modification needs a different approach (e.g. https://github.com/electron/electron/pull/24470#issuecomment-663166146)

Reply:
> Protecting against JS modification needs a different approach

Correct, but we can't do that yet.  This is IMO a reasonable middle ground that dramatically raises the bar of someone else turning this off / messing with it.  We also still don't have a _nice_ way of toggling `base::Feature` flags from JS, `appendSwitch` is not super ideal 😆